sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -663,3 +665,37 @@ def test_xfail_ignored_in_check_estimator():
     # Make sure checks marked as xfail are just ignored and not run by
     # check_estimator(), but still raise a warning.
     assert_warns(SkipTestWarning, check_estimator, NuSVC())
+
+
+def test_check_fit2d_1sample():
+
+    class MyEst(SVC):
+        # raises a bad error message when only 1 sample is passed
+        def fit(self, X, y):
+            if X.shape[0] == 1:
+                raise ValueError("non informative error message")
+
+    assert_raises_regex(
+        AssertionError,
+        "The error message should contain one of the following",
+        check_fit2d_1sample,
+        'estimator_name',
+        MyEst()
+    )
+
+
+def test_check_fit2d_1feature():
```

Comment:
That's exactly the same function as above?

Reply:
almost, but for different checks ;)
one checks samples the other checks features